README.md

```diff
@@ -1,2 +1,49 @@
 # lev-api-stress
-Stress testing for the `lev-api`.
+Stress testing for the [LEV API], based on the [Artillery.io] tool.
+
+
+## Running the test
+The performance tests can be started with the following command:
+```shell script
+./entrypoint.sh <path-to-config-file>
+```
+
+
+## Configuration
+There are two parts to the configuration:
+ - `./high-load.yml` - the main config file for [Artillery.io], containing:
+   - basic test setup
+   - security details
+   - config for the phases
```

Comment:
Worth explaining why there are different phases?

Reply:
Well there's currently only a single phase definition. We haven't made use of multiple phases yet.

Reply:
Once we make use of multiple phases, it would probably be worth adding something.

Reply:
Oh is the 'ramp-up' not a phase?

Reply:
It could be a phase, but so far Aaron and I haven't found it to be especially useful for our manual testing. If we were to build performance tests into CD, it would likely be useful at that stage.